core/storage/suggestion/gae_models.py

```diff
@@ -743,11 +709,10 @@ def get_reviewable_translation_suggestions_for_single_exploration(
             next_offset
         )
 
-    # TODO(sll): Transition the callsite to use the new method
+    # TODO(#18745): Transition the callsite to use the new method
```

Comment:
the callsite --> all callsites

In the issue, could you please explain more why we are doing this and how to find the callsites? Give a bit more explanation of what specific work has to be done as well (e.g. we need to prevent the frontend from requesting data for multiple exp IDs at a time, which shouldn't be necessary anyway).

Reply:
Done.